test/tools/unified-spec-runner/entities.ts

```diff
@@ -205,7 +195,7 @@ export class UnifiedMongoClient extends MongoClient {
       mongodbLogPath: logCollector,
       ...getEnvironmentalOptions(),
       ...(description.serverApi ? { serverApi: description.serverApi } : {})
-    });
+    } as any);
```

Comment:
Instead of casting this as `any`, can we use a ts-expect-error and in the message, link to the jira ticket?

We expect this to have a TS error until add `mongodbLogPath` to the client options.